frontend/src/app/Settings/Facility/Components/ManageDevices.scss

```diff
@@ -0,0 +1,24 @@
+.device-settings {
+  .results-dropdown {
+    top: 2.5em;
+    max-height: 12.1em;
+    min-width: 100%;
+    overflow-y: scroll;
```

Comment:
have you considered changing this to `overflow-y: auto;`
That will make the scrolling bar only show up when its needed, for example, it will not show up in the empty state

Reply:
Yes, but I ran into a layout issue that I had a hard time working around:

https://github.com/CDCgov/prime-simplereport/assets/27730981/593c4596-0815-4c8c-9c64-16e58ffb6b53

Revisiting this, though, I've learned about the `scrollbar-gutter: stable` CSS property that handles this way more cleanly:

https://github.com/CDCgov/prime-simplereport/assets/27730981/23373866-1289-4ec2-b967-bfb1ed6b89f3

Updated in [49b5f01](https://github.com/CDCgov/prime-simplereport/pull/6198/commits/49b5f01e1ed39431a0ffa6c855b52c7c5013ed44).

Reply:
oh neat! TIL that `scrollbar-gutter` exists